<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>670</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1240</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1240</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>在线考试系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_QuestionsManager">
   <property name="text">
    <string>题库管理</string>
   </property>
  </action>
  <action name="action_examctrl">
   <property name="text">
    <string>考试控制</string>
   </property>
  </action>
  <action name="action_makepaper">
   <property name="text">
    <string>试卷管理</string>
   </property>
  </action>
  <action name="action_subscore">
   <property name="text">
    <string>主观题批改</string>
   </property>
  </action>
  <action name="action_memmanager">
   <property name="text">
    <string>用户管理</string>
   </property>
  </action>
  <action name="action_login">
   <property name="text">
    <string>登录</string>
   </property>
  </action>
  <action name="action_mainmenu">
   <property name="text">
    <string>主菜单</string>
   </property>
  </action>
  <action name="action_config">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="action_scomanage">
   <property name="text">
    <string>成绩管理</string>
   </property>
  </action>
  <action name="action_Qt">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="action_program">
   <property name="text">
    <string>关于本程序</string>
   </property>
  </action>
  <action name="action_inoutput">
   <property name="text">
    <string>导入导出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
